utils/messages.py

```diff
@@ -18,3 +18,20 @@ def embedded_stats(title, **stats):
         embed.add_field(name=name, value=stat, inline=True)
 
     return embed
+
+
+def embedded_matchday_results(matchday, matches):
+    """Build embedded message with matchday results"""
+    title = "Matchday " + matchday
+    embed = discord.Embed(title=title, color=0xDC052D)
+
+    for key, value in matches.items():
+        # where value is a list of of 2 tuples
+        # each tuple contains: ('team_name', 'score')
+        embed.add_field(
+            name=value[0][0] + "-" + value[1][0],
+            value=value[0][1] + "-" + value[1][1],
```

Comment:
There may be a cleaner way of joining the items in the tuple rather than indexing.  Indexing can be difficult to understand if you don't know the exact structure of the tuple being passed in.  Maybe we could look into using an iterator with `.join()`?

Reply:
Great idea. Can you help me implement this?